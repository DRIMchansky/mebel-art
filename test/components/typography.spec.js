import React from 'react'
import renderer from 'react-test-renderer'
import Typography from '../../src/components/typography'

describe('Typography', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Typography />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
