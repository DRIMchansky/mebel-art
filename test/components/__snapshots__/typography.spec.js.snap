// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Typography renders correctly 1`] = `
.c0 {
  color: #212529;
  font-weight: 400;
}

.c0 h1 {
  font-size: 1.3rem;
}

.c0 h2 {
  font-size: 1.2rem;
}

.c0 h3,
.c0 h4,
.c0 h5,
.c0 h6 {
  font-size: 1.1rem;
}

.c0 p {
  font-size: 0.9rem;
  line-height: 1.4;
}

<div
  className="c0"
/>
`;
